core/crypto/pedersen_hash.go

```diff
@@ -1,85 +1,28 @@
 package crypto
 
 import (
-	"math/big"
-
 	"github.com/NethermindEth/juno/core/felt"
-	starkcurve "github.com/consensys/gnark-crypto/ecc/stark-curve"
 	"github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
+	pedersenhash "github.com/consensys/gnark-crypto/ecc/stark-curve/pedersen-hash"
```

Comment:
how about `gnark` instead of `pedersenhash`?

Reply:
I think `gnark` would have been appropriate if we were importing `github.com/consensys/gnark-crypto`. Imo, `fp` could as well be `gnark` since they are both subsets of `gnark`. 

Maybe we could do `hash`? Then, say `hash.Pedersen()` and `hash.PedersenArray()`. That feels better to me and it eliminates the  repetition of the word `pedersen` which I sense may be unsettling.

Reply:
Lets just keep it as is